lib/driip-settlement.js

```diff
@@ -47,12 +49,15 @@ class DriipSettlement {
         try {
             return await challengeContract.proposalNonce(address, ct, id);
         } catch (err) {
-            return null;
+            if (isRevertContractException(err)) 
+                return null;
+            
+            throw err;
```

Comment:
´throw new Error('Unable to get current proposal nonce: ' + err);`

Reply:
Used NestedError class to do the rethrow. The code copied from the crawler repo.